Bottom: acb90057a5c2227df4d480223275c85609da35d2
Top:    da1e70cd8fc10de520644a8dad2aa12fe55f82c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 23:44:38 +0100

draw.c: Pass clear_pins as true when calling DrawLayerCommon


---

diff --git a/src/draw.c b/src/draw.c
index 83080b3..fb29826 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1113,7 +1113,7 @@ DrawLayerCommon (LayerTypePtr Layer, const BoxType * screen, bool clear_pins)
 void
 DrawLayer (LayerTypePtr Layer, const BoxType * screen)
 {
-  DrawLayerCommon (Layer, screen, false);
+  DrawLayerCommon (Layer, screen, true);
 }
 
 /* ---------------------------------------------------------------------------
